src/azure-cli/azure/cli/command_modules/servicebus/_validators.py

```diff
@@ -116,3 +116,9 @@ def validate_subnet(cmd, namespace):
     else:
         raise CLIError('incorrect usage: [--subnet ID | --subnet NAME --vnet-name NAME]')
     delattr(namespace, 'vnet_name')
+
+
+def validate_rights(namespace):
+    if 'Manage' in namespace.rights:
+        if 'Listen' not in namespace.rights or 'Send' not in namespace.rights:
+            raise CLIError('Error : Assigning \'Manage\' to --rights requires \'Listen\' and \'Send\' to be included with. e.g. --rights Manage Send Listen')
```

Comment:
This seems like it should be a service-side error.

Reply:
@axisc, can you please share your view on this?